src/components/MetadataCard.tsx

```diff
@@ -39,14 +40,20 @@ export const MetadataTitle: React.FC<{ children: React.ReactNode }> = ({
 );
 
 interface ItemProps {
+  as?: PolymorphicAs;
   children: React.ReactNode;
   "data-cy"?: string;
 }
 
 export const MetadataItem: React.FC<ItemProps> = ({
+  as = "p",
   children,
   "data-cy": dataCy,
-}) => <Item data-cy={dataCy}>{children}</Item>;
+}) => (
+  <Item data-cy={dataCy} as={as}>
```

Comment:
This is cool! The only issue with this is that the margin from `:not(:last-of-type)` selector doesn't quite work as intended anymore due to the mix of `<p>` and `<div>` tags.

Maybe we can change `:not(:last-of-type)` -> `:not(:last-child)`